src/main/java/com/iota/iri/model/persistables/Hashes.java

```diff
@@ -31,6 +50,10 @@ public void read(byte[] bytes) {
         }
     }
 
+    /**
```

Comment:
better to document the interface

Reply:
This is a dummy implementation (should be abstract). Please delete the doc